CONTRIBUTING.md

```diff
@@ -1,4 +1,4 @@
-This repo is supposed to replace [/Starcounter/level1/tree/develop/src/BuildSystem/ClientFiles](https://github.com/Starcounter/level1/tree/develop/src/BuildSystem/ClientFiles). At this point it does not replace it yet, but it is the source of truth for it.
+This repo is the source of client files for [/Starcounter/level1/tree/develop/src/BuildSystem/ClientFiles](https://github.com/Starcounter/level1/tree/develop/src/BuildSystem/ClientFiles).
```

Comment:
The link is wrong, this should not exist anymore (I can see that not all ClientFiles are removed though, will create an issue for that: https://github.com/Starcounter/level1/tree/develop/src/BuildSystem/ClientFiles)

I think you should say something like, source of client files found in `%StarcounterBin%/ClientFiles`

Reply:
Issue has been created to remove remaining ClientFiles from Level1: https://github.com/Starcounter/level1/issues/4485

Reply:
Thanks, fixed.